docs/init.md

```diff
@@ -213,10 +213,11 @@ Traceback (most recent call last):
 ValueError: 'x' has to be smaller than 'y'!
 ```
 
-This example also shows of some syntactic sugar for using the {obj}`attrs.validators.and_` validator: if you pass a list, all validators have to pass.
+In this example, the list of validators passed to the `validator` argument, is syntactic sugar for the {obj}`attrs.validators.and_` validator: All validators in the list have to pass.
```

Comment:
I think I liked the former formulation better, since it isn't the list that is _syntactic sugar_, but the usage of the validator.

Reply:
Indeed that was the most difficult. 

Because I understood the example (that all validators in the list should pass), I could not understand "why I read this sentence?". 

It took me some time to grasp the connection between the list of validators and `attrs.validatos.and_`.

What about:

>This example is also showing an alternative use (i.e. syntactic sugar) for the {obj}`attrs.validators.and_` validator: 
All validators in the list have to pass.

Reply:
> What about:
> 
> > This example is also showing an alternative use (i.e. syntactic sugar) for the {obj}`attrs.validators.and_` validator:
> > All validators in the list have to pass.

I'd suggest: 

> This example shows a syntactic speciality for the {obj}`attrs.validators.and_` validator: if you pass a list, all validators in it have to pass.

Reply:
That is very good, I create a commit with it, thank you!